cscs-checks/libraries/math/trilinos_compile_run.py

```diff
@@ -2,20 +2,20 @@
 import reframe.utility.sanity as sn
 
 
+# FIXME: The 'dynamic' version of the tests gets stuck in compitation for
```

Comment:
```suggestion
# FIXME: The 'dynamic' version of the tests gets stuck in compilation for
```

Reply:
Why this is a `FIXME`? It's not a problem of the test. I guess `NOTE` fits better.